identify/README.md

```diff
@@ -133,3 +134,10 @@ clients only support initiating requests while some servers (only) support
 responding to requests. To prevent clients from initiating requests to other
 clients, which given them being clients they fail to respond, clients should not
 advertise `foo` in their `protocols` list.
+
+### signedPeerRecord
+
+This is a serialized [SignedEnvelope](https://github.com/libp2p/go-libp2p/blob/master/core/record/pb/envelope.proto) containing a [PeerRecord](https://github.com/libp2p/go-libp2p/blob/master/core/peer/pb/peer_record.proto),
+signed by the sending node. It contains the same addresses as the `listenAddrs` field, but in a form that lets us share authenticated addrs with other peers.
+
+This field was introduced in a backwards compatible manner (meaning that it is sent along with the `listenAddrs` field), therefore, it is optional and may be omitted by older implementations.
```

Comment:
I think we should add a paragraph like:

> If the signed peer record is present, implementations MUST use the data contained within it and ignore duplicated fields in the main identify message.

Reply:
Besides `listenAddrs`, are there are any other fields that may be present in the signed envelope?

Reply:
Not right now but the [routing records](https://github.com/libp2p/specs/blob/master/RFC/0003-routing-records.md) calls out it's future extensibility, for example the list of supported protocols.